### PR TITLE
[CU-3axvr5v] Add subsquid handlers and resolvers for picasso overview stats

### PR DESCRIPTION
**Pull request by andresmechali**
## Issue
https://app.clickup.com/t/3axvr5v

## Missing
Review CI

### CODE REVIEW

**Review by easteregg — APPROVED** [review]

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-3axvr5v Picasso Stats FE Integrations Subsquid Requirements](https://app.clickup.com/t/3axvr5v)

**Review by saadjhk — APPROVED** [review]

File: subsquid/Makefile [hunk 1]
```diff
@@ -44,5 +44,15 @@ up:
 down:
 	@docker-compose down
 
+rebuild:
+	rm -f -r lib
+	rm -f -r db/migrations
+	npx sqd db drop
```
Comment (saadjhk) [inline subsquid/Makefile:50]: can we remove these, as they are deprecated `npx sqd`

**Review by saadjhk — APPROVED** [review]

File: subsquid/schema.graphql [hunk 2]
```diff
@@ -1,44 +1,12 @@
...
-  quoteAssetId: BigInt!
-  quoteAssetAmount: BigInt!
-  blockNumber: BigInt!
-  spotPrice: String!
-  "Optional: Only certain transaction types have fees charged by Pablo. Does NOT include the collected extrinsic execution fee."
-  fee: String!
-  "Unix timestamp in ms"
-  receivedTimestamp: BigInt!
+  "Last event involving account"
+  eventId: String!
```
[the remark this answers is not in the document]
Comment (andresmechali) [reply subsquid/schema.graphql:5]: no, it's the id of the chain event

File: subsquid/Makefile (hunk 1 above)
Comment (andresmechali) [reply subsquid/Makefile:50]: yes, well spotted

**Review by vimukthi-git — APPROVED** [review]
LGTM :rocket:

**Comment by andresmechali** [comment]
@mergify refresh

**Pull request merged**